spec/features/admin/admin_deletes_comment_spec.rb

```
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+feature 'Admin deletes comment - ', %(
+  As an admin
+  I want to delete a comment
+  So that I can cull the comment population
+) do
+
+  scenario 'Admin should be able to delete comments' do
+    sign_in_admin
+    comment = FactoryGirl.create(:comment)
+    visit tool_path(comment.review.tool)
+    click_button("Delete comment #{comment.id}")
+    expect { Comment.find(comment.id) }.
+      to raise_error { "Couldn't find Tool with 'id'=#{comment.id}" }
```

Searching for the comment that was just deleted should raise an error, proving that the delete button is working.